fiftyone/core/odm/database.py

```
@@ -1795,8 +1904,39 @@ def _get_result_ids(conn, dataset_dict):
+def _get_generated_dataset_name_for_saved_view_if_any(view_doc):
+    # By convention all views that use generated datasets store its
+    # name in a `_state.name` parameter
+    for view_stage_str in view_doc.get("view_stages", []):
+        try:
+            view_stage_dict = json_util.loads(view_stage_str)
+            view_stage_kwargs = dict(view_stage_dict.get("kwargs", {}))
+            _state = view_stage_kwargs.get("_state", {})
+            name = _state.get("name", None)
+            if name:
+                return name
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)